time/src/main/java/io/spine/time/Now.java

```diff
@@ -178,7 +187,7 @@ private ZonedDateTime now() {
         Timestamp time = Time.currentTime();
         Instant instant = InstantConverter.reversed()
                                           .convert(time);
-        checkNotNull(instant);
+        requireNonNull(instant);
```

Comment:
Why not `checkNotNull`?

Reply:
It causes warning because `instant` can be `null`, and `checkNotNull()` accepts non-nullable type.

Reply:
It's a bit weird. We're explicitly using `checkNotNull` to verify that `instant` is not `null` at a particular moment. I'd say it may be better to suppress the warning rather than having separate approaches.